src/main/java/at/hannibal2/skyhanni/utils/SkyBlockItemModifierUtils.kt

```diff
@@ -137,18 +138,27 @@ object SkyBlockItemModifierUtils {
     @Suppress("CAST_NEVER_SUCCEEDS")
     inline val ItemStack.cachedData: CachedItemData get() = (this as ItemStackCachedData).skyhanni_cachedData
 
+    val warnedAboutPetParseFailure: MutableSet<String> = mutableSetOf()
+    var lastWarnedParseFailure: SimpleTimeMark = SimpleTimeMark.farPast()
+
     fun ItemStack.getPetInfo(): PetInfo? {
+        val colorlessName = displayName.removeColor()
+        // Repo pets will always return null for PetInfo, don't even attempt to parse it
+        if (colorlessName.contains("→")) return null
```

Comment:
```suggestion
        if (colorlessName.contains("→") || colorlessName.contains("{LVL}")) return null
```
https://discord.com/channels/997079228510117908/1387472530989650121/1387472530989650121

Reply:
```
Extra data:
exception: 'No argument provided for a required parameter: parameter #2 exp of fun `<init>`(kotlin.String, kotlin.Boolean?, kotlin.Double,
```

That was set to default of 0.0 in this PR, so this still would fix that error - the arrow catcher was just a cherry on top - this is probably another good one to include though.